pages/projects/submit.tsx

```diff
@@ -7,6 +7,8 @@ import { useForm, SubmitHandler, Controller } from 'react-hook-form';
 import { useSession } from 'next-auth/react';
 import Router from 'next/router';
 import { motion } from 'framer-motion';
+import Lottie from "lottie-react";
+import Loader from "./Loader/98288-loading.json";
```

Comment:
can you put this json file inside public folder. and please rename the json file to loading.json

Reply:
yes I will change.